<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	
	font-family:century gothic;
	font-size:24px;
	background-color:#e0e5ff
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>351</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{background-color:#ffe045;}
QPushButton
{
border-radius:30px;
	background-color:rgb(203, 255, 203);
}
QPushButton:hover
{
background-color:rgb(140, 234, 255)
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="UserName_LineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Password_LineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sign_in_PushB">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>410</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Sign in</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Login_PushB">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>320</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:18px;
font-weight:bold</string>
     </property>
     <property name="text">
      <string>Welcome to ToDoList Andventure</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="invalid_password">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:15px;
color:red;
</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="invalid_username">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:15px;
color:red;
</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
